esmf_regrid/schemes.py

```diff
@@ -276,7 +275,7 @@ def _regrid_along_dims(regridder, data, dims, num_out_dims, mdtol):
     return result
 
 
-def _map_complete_blocks(src, func, dims, out_sizes):
+def _map_complete_blocks(src, func, active_dims, out_sizes, *args, **kwargs):
```

Comment:
@stephenworsley Is your inclusion of `*args` here speculative of future `func` API?

At the moment `*args` doesn't seem to be used ... just curious.

Reply:
yeah, I suppose this is speculative for the time being, though the idea here is that I wanted to match the signature used by `map_blocks`. I decided to preserve both args and kwargs for the sake of completeness.

Reply:
This also matches the pattern I used in iris here https://github.com/SciTools/iris/pull/5767 which preserve args for the same reasons.